docs/concepts-broker-load-balancing-quick-start.md

```diff
@@ -0,0 +1,594 @@
+```
+
+**Output**
+
+If there is no output, the operation is successful.
+
+4.3 Produce messages to the topic using the [pulsar-perf tool](./reference-cli-tools.md).
+
+**Input**
+
+```bash
+bin/pulsar-perf produce persistent://my-tenant-1/my-namespace-1/my-topic-1 -r 400 -bm 1 -mk random
+```
+
+**Output**
+
+The output shows that messages have been produced.
+
+```bash
+2023-07-21T05:08:24,261+0000 [main] INFO org.apache.pulsar.testclient.PerformanceProducer - JVM args [-Dlog4j.configurationFile=log4j2.yaml, -Djava.net.preferIPv4Stack=true, --add-opens=java.base/sun.net=ALL-UNNAMED, --add-opens=java.base/java.lang=ALL-UNNAMED, -Dpulsar.allocator.exit_on_oom=true, -Dio.netty.recycler.maxCapacityPerThread=4096, -Dpulsar.log.appender=Console, -Dpulsar.log.level=info, -Dpulsar.log.root.level=info, -Dpulsar.log.immediateFlush=false, -Dpulsar.log.dir=/pulsar/logs, -Dpulsar.log.file=pulsar-perftest.log] 2023-07-21T05:08:24,305+0000 [main] INFO org.apache.pulsar.testclient.PerformanceProducer - Netty max memory (PlatformDependent.maxDirectMemory()) 1 GB 2023-07-21T05:08:24,305+0000 [main] INFO org.apache.pulsar.testclient.PerformanceProducer - JVM max heap memory (Runtime.getRuntime().maxMemory()) 1 GB 2023-07-21T05:08:24,409+0000 [main] INFO org.apache.pulsar.testclient.PerformanceProducer - Starting Pulsar perf producer with config: { 
+...
+```
+
+### Step 5. Split bundles
+
+5.1 Open another tab in the terminal (i.e., terminal tab 2) and execute the following commands.
+
+5.2 Enter the container of broker 2.
+
+**Input**
+
+```bash
+docker exec -it broker-2 bash
+```
+
+**Output**
+
+If there is no output, the operation is successful.
+
+5.3 Verify whether the bundles are split after 5 minutes. It takes a little time for Pulsar to split bundles.
+
+:::tip
+
+- The automatic bundle splitting is **enabled by default**. To disable it, update the following configurations to false in the broker.conf file. 
+  
+    - [loadBalancerAutoBundleSplitEnabled](https://github.com/apache/pulsar/blob/69d7a2bf14555f11a716a9545c5cf391d8179a27/conf/broker.conf#L1278)
+  
+    - [loadBalancerAutoUnloadSplitBundlesEnabled](https://github.com/apache/pulsar/blob/69d7a2bf14555f11a716a9545c5cf391d8179a27/conf/broker.conf#L1281C4-L1281C4) 
+
+- The default bundle splitting algorithm is `range_equally_divide`. You can change it to other [bundle splitting algorithms](./concepts-broker-load-balancing-concepts.md#bundle-splitting-algorithms) by updating [defaultNamespaceBundleSplitAlgorithm](https://github.com/apache/pulsar/blob/69d7a2bf14555f11a716a9545c5cf391d8179a27/conf/broker.conf#L1321C18-L1321C18) in the broker.conf file. 
+
+- For bundle splitting thresholds, you can set more configurations in the broker.conf file. Any existing bundle that exceeds any of the thresholds is a candidate to be split. 
```

Comment:
I'll add the configuration link once that link is available (the configuration doc PR is WIP).